contractcourt/chain_arbitrator.go

```diff
@@ -678,6 +693,34 @@ func (c *ChainArbitrator) Start() error {
 	return nil
 }
 
+// getBestBlock gets the best block height that the arbitrator has knowledge of
+// querying our chain backend directly if this value is not yet set.
+func (c *ChainArbitrator) getBestBlock() (int32, error) {
+	c.bestBlockLock.Lock()
+	height := c.bestBlock
+	c.bestBlockLock.Unlock()
+
+	if height != 0 {
+		return height, nil
+	}
+
+	_, height, err := c.cfg.ChainIO.GetBestBlock()
+	if err != nil {
+		return 0, err
+	}
+
+	c.setBestBlock(height)
```

Comment:
Could it happen that if `ChainIO.GetBestBlock()` is taking too long then we'll override a recently set best block with a smaller one here?

Reply:
Since we'll only need to go to `GetBestBlock` on startup of chain arb (only time when we'll have a 0 height), I don't think that this will be a problem? But I can add a check to only update if > current height to be safe?